setup.py

```diff
@@ -2,6 +2,9 @@
 
 __version__ = '1.0.1'
 
+# Dependencies to make Amundsen work with Apache Atlas
+atlas = ['atlasclient>=0.1.2']
```

Comment:
Just wonder how much of dependency `atlasclient` would bring. Brining both neo4j and atlas dependencies would be unnecessary but if it's light it should be fine.

Reply:
agree

Reply:
but I think @verdan brought that the atlasclient dependency is not very well maintained(https://amundsenworkspace.slack.com/archives/CGFBVT23V/p1552426276000400?thread_ts=1551375775.001000&cid=CGFBVT23V)?

Reply:
at the moment, yes we don't know what else would be needed to make `atlas` work with amundsen, therefore, I thought to keep it separate for now, and will combine neo4j and atlasclient, if this is the only dependency after the whole integration of atlas with amundsen. Thoughts? I can add a TODO here. 

@feng-tao true. The problem with atlasclient package is mainly the documentation, with incorrect code snippets/examples. So you need to hit and try or debug the functions in order to get the proper results. But this package also has some nice serializers to convert relations into JSON properly. Therefore I suggest to use this package and I will make sure to open PRs to update the documentation and refactor the library along the way.